src/app/pages/blog/blog-context.js

```diff
@@ -73,12 +73,4 @@ function useLocationSynchronizedToPath() {
     };
 }
 
-export function BlogContextProvider({children}) {
-    const value = useLocationSynchronizedToPath();
-
-    return (
-        <BlogContext.Provider value={value} children={children} />
-    );
-}
-
 export default BlogContext;
```

Comment:
I've found it's convenient to export a `useBlogContext` hook so that consumers only need a single import and can't muck it up

Reply:
Currently, they `import BlogContext` and call `React.useContext(BlogContext)`. Is your idea to wrap the `useContext` in its own hook?

Reply:
yah that's what I often do.  It just seems cleaner to me to have a single `useBlogContext()` call vs `React.useContext(BlogContext)`    With the later you have to import from both the BlogContext and useContext. not a big deal either way of course